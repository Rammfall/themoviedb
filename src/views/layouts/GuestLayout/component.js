import React from 'react'
import PropTypes from 'prop-types'
import { Col, Layout, Row } from 'antd'

const GuestLayout = ({ children }) => (
  <div className="center">
    <Layout>
      <Layout.Content>
        <Row
          type="flex"
          justify="center"
        >
          <Col>{children}</Col>
        </Row>
      </Layout.Content>
    </Layout>
  </div>
)

GuestLayout.propTypes = {
  children: PropTypes.node.isRequired
}

export default GuestLayout
