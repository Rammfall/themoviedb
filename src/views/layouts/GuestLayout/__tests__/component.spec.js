import React from 'react'
import { shallow } from 'enzyme'

import GuestLayout from '../component'

describe('GuestLayout', () => {
  it('matches snapshot', () => {
    const component = shallow(<GuestLayout>Test</GuestLayout>)

    expect(component).toMatchSnapshot()
  })
})
