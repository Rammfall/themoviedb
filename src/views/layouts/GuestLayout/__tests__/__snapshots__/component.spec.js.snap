// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GuestLayout matches snapshot 1`] = `
<div
  className="center"
>
  <Layout>
    <Content>
      <Row
        justify="center"
        type="flex"
      >
        <Col>
          Test
        </Col>
      </Row>
    </Content>
  </Layout>
</div>
`;
